pkg/controller/sparkapplication/controller.go

```diff
@@ -125,22 +128,27 @@ func newSparkApplicationController(
 		controller.metrics = newSparkAppMetrics(metricsConfig)
 		controller.metrics.registerMetrics()
 	}
+	namespaceFilter := util.GetNamespaceFilter(namespaceFilterConfig)
 
 	crdInformer := crdInformerFactory.Sparkoperator().V1beta2().SparkApplications()
-	crdInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+	crdEventHandlerFuncs := cache.ResourceEventHandlerFuncs{
 		AddFunc:    controller.onAdd,
 		UpdateFunc: controller.onUpdate,
 		DeleteFunc: controller.onDelete,
-	})
+	}
+	crdEventResourceHandlerFuncs := getEventResourceHandlerFuncs(crdEventHandlerFuncs, namespaceFilter)
```

Comment:
Please rename both the variable and function names to use `ResourceEvent` instead of `EventResource`.

Reply:
Done.